test/TritonGPU/loop-pipeline-trans.mlir

```diff
@@ -0,0 +1,212 @@
+// RUN: triton-opt %s -split-input-file -tritongpu-pipeline=num-stages=3 -canonicalize | FileCheck %s
```

Comment:
The test could be included in loop-pipeline.mlir

Reply:
Sounds good.